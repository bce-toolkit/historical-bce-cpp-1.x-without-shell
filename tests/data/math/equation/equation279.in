6 5
3 0 0 -1 0 0
12 2 0 -3 -2 0
1 0 -2 0 0 0
4 2 -8 0 -1 0
0 1 -3 0 0 0
# (NH4)3PO4+Ca(OH)2=Ca3(PO4)2+NH3+H2O
