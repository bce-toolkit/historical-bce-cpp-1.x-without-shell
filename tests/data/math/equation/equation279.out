1/3Xa
1/2Xa
1/6Xa
Xa
Xa
